javascript/ql/test/library-tests/ModelGeneration/ModelGeneration.expected

```diff
@@ -1,7 +1,19 @@
 typeModel
 | (reexport).func | reexport | Member[func] |
+| (return-this).FluentInterface | return-this | Member[FluentInterface] |
+| (return-this).FluentInterface.prototype | (return-this).FluentInterface | Instance |
+| (return-this).FluentInterface.prototype | (return-this).FluentInterface.prototype.bar | ReturnValue |
+| (return-this).FluentInterface.prototype | (return-this).FluentInterface.prototype.baz | ReturnValue |
```

Comment:
Why is there not this row?

```
| (return-this).FluentInterface.prototype | (return-this).FluentInterface.prototype.foo | ReturnValue |
```

Reply:
Huh, good question. The flow is captured by the summary model so it shouldn't be a problem, but it is a bit strange that it's generated for `bar,baz` but not `foo`.

Reply:
I don't quite know what to say around this. I think it would be nice if we could at least explain why it's happening, but I might be a little too cautious around this.

Reply:
I figured it out, turns out this was due to a bug in API graphs. See https://github.com/github/codeql/pull/15386/commits/3c885f3969de522c625d4490140ae585b25a4576.

Nice catch, btw!